packages/tables/src/styled/StyledCell.ts

```diff
@@ -21,18 +20,68 @@ export interface IStyledCellProps {
   hasOverflow?: boolean;
   /** The width of the cell */
   width?: string | number;
+  size: SIZE;
 }
 
-export const StyledCell = styled.td.attrs<IStyledCellProps>(props => ({
+const truncatedStyling = css`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`;
+
+const sizeStyling = (props: IStyledCellProps & ThemeProps<DefaultTheme>) => {
+  let padding = `${props.theme.fontSizes.xs} ${props.theme.fontSizes.sm}`;
+  let verticalPadding;
+  let rightPadding;
+  let width = props.width;
+  let height;
+
+  if (props.hasOverflow) {
+    padding = '0';
+    width = '2em';
+    height = 'inherit';
+  } else if (props.size === 'large') {
+    verticalPadding = props.theme.fontSizes.xl;
+  } else if (props.size === 'small') {
+    verticalPadding = '6px';
+  }
+
+  if (props.isMinimum) {
+    if (props.theme.rtl) {
+      rightPadding = `${props.theme.space.base * 3}px`;
+    }
+
+    rightPadding = '0';
```

Comment:
[nit] calculating `horizontalPadding` would feel more intuitive throughout (vs. `rightPadding`) and then apply below based on RTL/not-RTL.

Reply:
The `padding-right` logic is determined by a combination of `rtl` and `isMinimum` and should only override a single left-right value with the parent `padding` declaration. I'm not sure combining these values would be clearer.

Reply:
Note that the padding is currently incorrect for RTL. There should be a left pad of 10px for non-RTL and a right pad of 10px for RTL. Once you correct the logic, it will make more sense to name this variable `horizontalPadding`. See https://austin-next.netlify.com/tables/?#selectable for the issue.

Reply:
Updated to `horizontalPadding`